src/runtime/String.h

```diff
@@ -165,6 +165,13 @@ class String : public PointerValue {
         m_bufferAccessData.hasSpecialImpl = false;
     }
 
+    enum NormalizationForm {
+        NFC,
+        NFD,
+        NFKC,
+        NFKD
```

Comment:
I think you can move this enum to builtinStringNormalize function.
Because there is no way to use this enum in other place

Reply:
Yes, this enum is used only by builtinStringNormalize, so I moved it into the function as you commented.